contents/teams/marketing/objectives.mdx

```diff
@@ -1,92 +1,71 @@
-## Product focus: Web analytics
-
-- **Rationale:** We want to focus most of our marketing efforts around a single product, giving the team greater focus. We're choosing web analytics because it's recently out of beta and has strong adoption already.
-
-- **Owned by:** Lior and Ian
-
-- **Contributors:** Andy and others where needed
+## Newsletter
 
+- **Rationale:** Our newsletter is the centerpiece of our content. We have published really good content and want to continue to do this.
+- **Owned by:** Ian
+- **Contributors:** *in the style of Gary Oldman in Leon* EVERYONE!
 - **What we're shipping:**
-  - Great web analytics docs
-  - Comparisons for main competitors
-  - Alternatives guides for main competitors
-  - Better privacy docs and advice (GDPR consent etc.)
-  - Proper web vs product analytics guide
-  - Loads of relevant tutorials and SEO content
-  - Social ads for web analytics
-
+ - Publish at least one newsletter every two weeks
+ - Test a new format that can potentially help us publish more regularly
+ - Every issue gets a James Twitter post and a LinkedIn post
+ - Bijan and Danilo publish a newsletter
 - **We'll know we're successful when:**
-  - We see increasing usage of web analytics
-  - More people choosing web analytics at onboarding
-  - Growing word of mouth for PostHog as a web analytics tool (brand mentions)
-
-## Content focus: Newsletter
+  - Our subscriber growth rises to over 8k per month.
+  - Everyone feels confident in shipping a newsletter issue. 
+  - We see a significant growth in Twitter and LinkedIn as sources of traffic.
 
-- **Rationale:** We've built a successful newsletter, but we need to aim higher. We want to ship more ambitious and original articles. Our ultimate long-term goal is people should talk about Product for Engineers in the same, or similar, terms as Lenny's Newsletter. A "default" newsletter that is synonmous with quality.
-
-- **Owned by:** Andy
-
-- **Contributors:** Ian, Lior, James Hawkins and anyone else who wants to get involved
+## Podcast
 
+- **Rationale:** As part of a marketing flywheel, a podcast stood out as an unexplored channel. And serves as a potential stepping stone to additional types of video content.
+- **Owned by:** Bijan
+- **Contributors:** James Hawkins
 - **What we're shipping:**
-  - Bar raising issues of the newsletter
-  - Working toward a higher frequency of issues
-  - Potentially new newsletter formats
-  - Andy attempting to become a LinkedIn influencer #youreathoughtleadernowharry
-
+  - Publish 10 podcast episodes
```

Comment:
```suggestion
  - Find a podcast producer to aid in production and editing (audio and video version).
  - Record 10 podcast episodes. Publish the best ones once we've settled on a format / branding etc.
```

@bijanbwb @jamesefhawkins I think this is a better framing. Given we need to experiment and test first, I think we should assume the firs two or three should be tests that we won't necessarily publish.

Reply:
Sounds good to me.

I was hesitant to outsource the podcast editing and tasks, because Riverside makes it relatively straightforward to do. But if we're planning to release a podcast every week then you're right it could get time-consuming and tedious.

I spent this morning editing a test podcast and it's actually pretty fun. There's a ton of features that Riverside gives us like auto-generating show notes, removing "uh" and "um" (which I apparently say every sentence), generating social clips, etc.

So yeah I think it'll be good for us to have those skills and capabilities in-house. But if we want to ramp up and keep a consistent weekly publishing schedule then it's not a bad idea to outsource to a free-lancer.

Reply:
Ok, let's drop the "producer" bit for now then and we can just keep it as something to think about as we go. If it ends up taking more time, we can adjust accordingly. 

RE: frequency, I think that's TBD really. In my head, it would make a lot of sense for podcast to be bi-weekly and interchange either with newsletter.